src/utils/string/StringUtils.js

```diff
@@ -345,7 +345,7 @@ const getChannelListInfo = (channel, selfSignUserId, selfAnonUserId) => {
     channelImage = DEFAULT_PROFILE_PIC_PATH;
   }
 
-  if (channel?.type === 'topics') {
+  if (channel?.type === 'topics' || channel?.type === 'topicinvitation') {
     channelImage = channel?.image || channel?.channel_image;
     channelName = channel?.name;
     if (!channelName || channelName === '') {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for handling 'topicinvitation' type is the same as 'topics'. This is a good example of code reuse and maintainability. However, consider creating constants for these string literals ('topics', 'topicinvitation') to avoid potential typos in future and improve readability.
```diff
- if (channel?.type === 'topics' || channel?.type === 'topicinvitation') {
+ const TOPICS_TYPE = 'topics';
+ const TOPIC_INVITATION_TYPE = 'topicinvitation';
+ if (channel?.type === TOPICS_TYPE || channel?.type === TOPIC_INVITATION_TYPE) {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->